#
# Use standard (ISO) date and time format
#
Index: xscreensaver-git/driver/XScreenSaver.ad.in
===================================================================
--- xscreensaver-git.orig/driver/XScreenSaver.ad.in
+++ xscreensaver-git/driver/XScreenSaver.ad.in
@@ -103,7 +103,9 @@
 
 
 ! The strftime(3) format string for printing the time on the password dialog.
-*dateFormat:		%I:%M %p, %a %b %e
+*dateFormat:		%Y-%m-%d %H:%M:%S
+! Original upstream format:
+! *dateFormat:		%I:%M %p, %a %b %e
 ! DD MMM:
 ! *dateFormat:		%I:%M %p, %e %b (%a)
 ! 12 hour time only:
