#
# For reproducible builds. Bug #819595
#
Index: xscreensaver-git/hacks/Makefile.in
===================================================================
--- xscreensaver-git.orig/hacks/Makefile.in
+++ xscreensaver-git/hacks/Makefile.in
@@ -856,8 +856,8 @@ cwaves:		cwaves.o	$(HACK_OBJS) $(COL)
 
 m6502.h:
 	@echo "building m6502.h from $(srcdir)/images/m6502/*.asm"; \
-	UTILS_SRC="$(UTILS_SRC)" \
-	$(srcdir)/m6502.sh m6502.h $(srcdir)/images/m6502/*.asm
+	find $(srcdir)/images/m6502/ -name '*.asm' | LC_ALL=C sort | \
+	  UTILS_SRC="$(UTILS_SRC)" xargs $(srcdir)/m6502.sh m6502.h
 
 m6502.o:	m6502.h
 m6502:		m6502.o		asm6502.o $(HACK_OBJS) $(ATV) $(PNG)
